<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sm3_sign</class>
 <widget class="QMainWindow" name="sm3_sign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件签名</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_sm3sign_sign">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>签名并保存</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/-_另存为.png</normaloff>:/icons/icons/-_另存为.png</iconset>
    </property>
   </widget>
   <widget class="CustomLineEdit" name="lineEdit_dir_raw_file">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sm3sign_cho_raw_file">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择原文件</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/-_打开项目.png</normaloff>:/icons/icons/-_打开项目.png</iconset>
    </property>
   </widget>
   <widget class="CustomLineEdit" name="lineEdit_dir_pri_file">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sm3sign_cho_pri_file">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择私钥文件</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/-_打开项目.png</normaloff>:/icons/icons/-_打开项目.png</iconset>
    </property>
   </widget>
   <widget class="CustomLineEdit" name="lineEdit_dir_signed_file">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sm3sign_cho_signed_dir">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择保存路径</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/-_打开项目.png</normaloff>:/icons/icons/-_打开项目.png</iconset>
    </property>
   </widget>
   <widget class="CustomLineEdit" name="lineEdit_signer_id">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>123hello#@</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>签名者标识：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">customlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
